api_callable.go

```diff
@@ -0,0 +1,98 @@
+package gax
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"time"
```

Comment:
It seems that import is not managed by gofmt then?  See  https://github.com/golang/go/wiki/CodeReviewComments#imports for the recommended style (and it says another command, goimports, to fix the import styles...)